### PR TITLE
Hotfix/several minor modifications

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/LIT-241
https://jira.itkdev.dk/browse/LIT-243
https://jira.itkdev.dk/browse/LIT-244
https://jira.itkdev.dk/browse/LIT-251
https://jira.itkdev.dk/browse/LIT-253
https://jira.itkdev.dk/browse/LIT-254
https://jira.itkdev.dk/browse/LIT-255

Have fun!

### CODE REVIEW

File: web/themes/custom/litteratursiden/sass/_front-page.scss [hunk 1]
```diff
@@ -631,4 +631,41 @@ body {
 
 }
 
+// What a bloody mess! Nothing to do but to continue down this dark path.
+// ITK Overrides.
```
Comment (cableman) [inline web/themes/custom/litteratursiden/sass/_front-page.scss:635]: My eyes ..... arghhh

File: web/themes/custom/litteratursiden/templates/node/node--article--spot.html.twig [hunk 2]
```diff
@@ -73,6 +73,11 @@ not node.isPublished() ? 'is-unpublished',
 view_mode ? view_mode|clean_class,
 'clearfix',
 ] %}
+{% set use_teaser = FALSE %}
+{% if node.field_teaser[0].value is not empty %}
+  {% set use_teaser = TRUE %}
+{% endif %}
```
Comment (cableman) [inline web/themes/custom/litteratursiden/templates/node/node--article--spot.html.twig:79]: I'am not strong in twig land... but would this not normally be in an preprocess function or the like i D8?

Comment (cableman) [reply web/themes/custom/litteratursiden/templates/node/node--article--spot.html.twig:79]: The same question in the other `spot` twig files?

File: web/themes/custom/litteratursiden/templates/node/node--book--slick-item.html.twig [hunk 3]
```diff
@@ -104,4 +104,19 @@ view_mode ? view_mode|clean_class,
...
 
   </div>
+  <div class="hover-content">
+    <a class="link-wrapper" href="{{ url }}"></a>
+    <h2{{ title_attributes }}>
+      {{ label }}
+    </h2>
+    {% if content.field_book_body|render %}
+      <div class="description book">{% set text = content.field_book_body|render|striptags|raw|trim %}
+        {{ (text|length > 180 ? text|slice(0, 181)|split(' ')|slice(0, -1)|join(' ') ~ '…' : text)|raw }}
```
Comment (cableman) [inline web/themes/custom/litteratursiden/templates/node/node--book--slick-item.html.twig:114]: I see the black magic... but would this take word boundary into account? And UTF-8 chars etc.

File: web/themes/custom/litteratursiden/templates/node/node--article--spot.html.twig (hunk 2 above)
Comment (martinyde) [reply web/themes/custom/litteratursiden/templates/node/node--article--spot.html.twig:79]: I'm not sure there d8 best practices that determine if a variable should be set in the twig layer or a preprocess function.

File: web/themes/custom/litteratursiden/templates/node/node--book--slick-item.html.twig (hunk 3 above)
Comment (martinyde) [reply web/themes/custom/litteratursiden/templates/node/node--book--slick-item.html.twig:114]: Word boundary is not considered, but i believe UTF-8 is covered

**Pull request merged**